app/assets/js/components/Work/Tabs/Structure/Structure.jsx

```diff
@@ -203,6 +244,13 @@ const WorkTabsStructure = ({ work }) => {
             </>
           )}
 
+          <Button
+            onClick={() => setIsBehaviorModalVisible(true)}
+            disabled={isEditing || isReordering}
+          >
+            <span>Behavior</span>
```

Comment:
I didn't catch this until now... The user may not be aware of current behavior value  unless they expand the modal. 

Is it easy to make the value of the current behavior be the string in this span: **Individuals**, **Continuous**, or **Paged**? It could have a down caret to help make the button appear to have additional options. This is a tough UX thing as we don't have a convention for this type of dropdown in this area.